pkg/controller/cassandracluster/generator.go

```diff
@@ -250,8 +250,10 @@ func generateCassandraStatefulSet(cc *api.CassandraCluster, status *api.Cassandr
 	rollingPartition := cc.GetRollingPartitionPerRacks(dcRackName)
 	terminationPeriod := int64(api.DefaultTerminationGracePeriodSeconds)
 	var annotations = map[string]string{}
+	var tolerations = []v1.Toleration{}
 	if cc.Spec.Pod != nil {
 		annotations = cc.Spec.Pod.Annotations
```

Comment:
We should test for spec.pod.toleration and spec.pod.annotation separately

Reply:
Agreed, but the test is currently written to test the whole statefulset generation, thus why i added this here. What would you suggest to be tested separatly? What should be duplicated/kept/taken out of the current test?